adminpages/license.php

```diff
@@ -30,15 +30,15 @@
 	require_once( PMPRO_DIR . '/adminpages/admin_header.php' );
 } ?>
 	<div class="about-wrap">
-		<h2><?php _e('Paid Memberships Pro Support License', 'paid-memberships-pro' );?></h2>
+		<h2><?php esc_html_e('Paid Memberships Pro Support License', 'paid-memberships-pro' );?></h2>
 
 		<div class="about-text">
 			<?php if ( is_wp_error( $pmpro_license_check ) ) { ?>
 				<p class="pmpro_message pmpro_error"><strong><?php echo esc_html( sprintf( __( 'There was an issue validating your license key: %s', 'paid-memberships-pro' ), $pmpro_license_check->get_error_message() ) );?></strong> <?php _e('Visit the PMPro <a href="https://www.paidmembershipspro.com/login/?redirect_to=%2Fmembership-account%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dmembership-account%26utm_content%3Dkey-not-valid" target="_blank">Membership Account</a> page to confirm that your account is active and to find your license key.', 'paid-memberships-pro' );?></p>
```

Comment:
@ideadude these translated strings with the full HTML is pretty tricky, we need to fix these _blank target links to use the `rel` set as noopener/noreferrer for [security purposes](https://web.dev/external-anchors-use-rel-noopener/). Should we just change it here or leave it for future work to be done to decouple the HTML from the translation string?